strings/anagram/isanagram.go

```diff
@@ -0,0 +1,41 @@
+/*isanagram.go
+description: Checks if two given strings are anagram or not
+details: An anagram is a word or phrase that's formed by rearranging the letters of another word or phrase
+Reference: https://www.geeksforgeeks.org/pangram-checking/
+Author : Venkatasubramanian */
+
+package anagram
```

Comment:
```suggestion
// isanagram.go
// description: Checks if two given strings are anagram or not
// details: An anagram is a word or phrase that's formed by rearranging the letters of another word or phrase
// Reference: https://www.geeksforgeeks.org/pangram-checking/
// Author : Venkatasubramanian

// Package anagram does something <--- put your explanation here right on top of the package declaration
package anagram
```

Reply:
The multi line comment was not what @raklaptudirm was referring to in the previous review.

Reply:
Also, you referenced `pangram-checking` from geeksforgeeks but that is a completely different algorithm. Please check your references properly before committing them.